cmd/containerd-shim-runhcs-v1/start.go

```diff
@@ -171,7 +172,7 @@ The start command can either start a new shim or return an address to an existin
 		if err := shim.WritePidFile(filepath.Join(cwd, "shim.pid"), pid); err != nil {
 			return err
 		}
-		if err := shim.WriteAddress(filepath.Join(cwd, "address"), address); err != nil {
+		if err := writeAddress(filepath.Join(cwd, "address"), address); err != nil {
```

Comment:
we're ignoring the [`bootstrap.json` file](https://github.com/containerd/containerd/pull/9233) wholesale for now, right?